<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathPlanningSettingDialog</class>
 <widget class="QDialog" name="PathPlanningSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>规划设定器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>起始位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta1">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="iniTheta7"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>末端位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="endTheta7"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PathPlanningSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PathPlanningSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
